packages/giselle-engine/src/core/generations/generate-image.ts

```diff
@@ -253,6 +255,7 @@ export async function generateImageWithOpenAI({
 		name: "ai-sdk/openai",
 		metadata: telemetry?.metadata,
 		input: { messages },
+		tags: ["deprecated"],
```

Comment:
to distinguish this bare-instrumented trace with the newly introduced trace instrumented using NextGiselleEngine